src/pseudopeople/interface.py

```diff
@@ -97,6 +99,33 @@ def _generate_dataset(
     return noised_dataset
 
 
+def validate_source_compatibility(source):
+    # TODO: Clean this up w/ metadata
+    changelog = source / "CHANGELOG.rst"
+    if changelog.exists():
+        with open(changelog, "r") as file:
+            first_line = file.readline()
+        version = first_line.split("**")[1].split("-")[0].strip()
+        if version > "1.4.2":
```

Comment:
This won't be correct in general, e.g. it will think "1.10.2" or "1.4.10" is less than "1.4.2". You might consider using https://packaging.pypa.io/en/latest/version.html#packaging.version.parse

Reply:
Will add a ticket to make this better

Reply:
If we don't fix this for the Yellow Release, we should make a note somewhere not to release any data versions that would break this! Because even though the user would still get an error in the corner case I mentioned, the error would tell them to do exactly the wrong thing -- redownload the data, when what they need to do is update pseudopeople.

Reply:
https://python-semver.readthedocs.io/en/latest/usage/compare-versions.html

Reply:
This compare is wrong, and we shouldn't knowingly release with a bug if it's avoidable. If we don't want to add a new package to the setup.py, you could do something quick/dirty but correct for our purposes like [this](https://stackoverflow.com/a/11887825/4514152)

Reply:
Are there Jira tickets for these TODOs?

Reply:
There are enough of these compares on version it should get a fix IMHO.

Reply:
Ok done